dvc/repo/__init__.py

```diff
@@ -5,7 +5,6 @@
 from typing import TYPE_CHECKING
 
 from funcy import cached_property, cat
-from git import InvalidGitRepositoryError
```

Comment:
Loading `git` module is slow, and since nowadays we have backends, it is taken care by it.